synapse/storage/schema/common/delta/59/13schema_compat_version.sql

```diff
@@ -0,0 +1,17 @@
+/* Copyright 2021 The Matrix.org Foundation C.I.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+-- populate schema_compat_version for the first time.
+INSERT INTO schema_compat_version(compat_version) VALUES (59);
```

Comment:
Would it make sense to have this be defined in code next to `SCHEMA_VERSION` and updated when we check schema versions (ensuring it only increases)? It just feels a bit odd to me that we're defining these in two separate places, is all.

Reply:
ok, this is now done.